src/Engine/ProtoCore/Core.cs

```diff
@@ -54,6 +54,10 @@ public class Options
     {
         public Options()
         {
+            // Execute using new graphnode dependency
+            DirectDependencyExecution = true;
+            LHSGraphNodeUpdate = !DirectDependencyExecution;
```

Comment:
Oh i had to revert this, the reason was LHSGraphNodeUpdate must always be false if we are executing DirectDependency and if we want to test the orginal behavior (dynamic update), LHSGraphNodeUpdate  must be true. This is only for testing and both of these flags will be removed if we no longer want to support dynamic update